### ISSUE TITLE
Cannot build project using gradle-5.0-rc-2

### ISSUE THREAD

**Issue by RobinMaronierY** [apollographql/apollo-android#1106]
**[Apollo Android version 1.0.0-alpha2]**
I'm trying to migrate from gradle 4.10.2 to gradle 5.0 RC, I'm using Android Plugin '_3.2.1_'.
I do not know if the problem is related by gradle 5.0 RC, but my project build failed at the execution of '_generateDevDebugApolloClasses_'
--> `> java.io.FileNotFoundException: [...]/build/generated/source/apollo/generatedIR/devDebug (Is a directory)`

Everything worked before the migration.
Any ideas ? Is Apollo 1.0.0-alpha compatible with gradle 5.0 RC ?

**Comment by lgawin** [comment]
According to https://github.com/gradle/gradle/issues/7732#issuecomment-438678230, in `Gradle-5.0-RC-x` there was a change in `SourceTask` and how out-of-date files are reported. Now together with outdated files, all enclosing directories/folders are reported as well.

So for some sample android project instead of just one call (as with `gradle-4.x`):
```
/…/sample/app/build/generated/source/apollo/generatedIR/debug/src/main/graphql/com/example/DebugAPI.json
```
when using `gradle-5.0-rc-x`, there’re 7  calls with paths:
```
/.../sample/app/build/generated/source/apollo/generatedIR/debug
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src/main
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src/main/graphql
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src/main/graphql/com
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src/main/graphql/com/example
/.../sample/app/build/generated/source/apollo/generatedIR/debug/src/main/graphql/com/example/DebugAPI.json
```
To make the plugin work correctly, directories/folders need to ignored.

**Comment by fluxxion82** [comment]
How do we work around this until we get a new version of apollo with the fix? Can we just not use gradle-5.0-rc-2?
